3634/2/commands/save_files.rb

```diff
@@ -0,0 +1,43 @@
+require_relative 'image_loader'
+
+class SaveFiles
+  attr_reader :session, :session_key, :payload
+
+  def initialize(session, session_key)
+    @session = session
+    @session_key = session_key
+  end
+
+  def make_path
+    FileUtils.makedirs path
+  end
+
+  def path
+    @path ||= "user/#{session_key}/#{Time.now.strftime '%Y-%m-%d_%H:%M:%S'}"
+  end
+
+  def save_files
+    make_path
```

Comment:
How about making path in one method which will use existing or will create, but not splitting in two methods

Reply:
Deleted make_path method and moved its inners to save_files method